docs/admin/config.rst

```diff
@@ -77,9 +77,18 @@ AUTO_UPDATE
 
 .. versionadded:: 3.2
 
-Automatically update all repositories on daily basis. This can be useful if you
+Update all repositories on daily basis. This can be useful if you
 do not use :ref:`hooks` to update Weblate repositories automatically.
 
+.. note::
+
+    There are both boolean options or string due to backwards compatibility.
+
+Options are:
+* `"none"` - no daily updates
+* `"remote"` also `False` - update remotes only
+* `"full"` also `True` - update remotes and merge working copy
```

Comment:
Please use double backticks for literals, see https://www.sphinx-doc.org/en/latest/usage/restructuredtext/basics.html